openmeetings-db/pom.xml

```diff
@@ -120,6 +120,27 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
+		<dependency> <!-- temporary, powermock doesn't support junit5 -->
```

Comment:
copy/pasting is BAD :(
please move these dependencies to `parent` since they are required by more tan one component

Reply:
well I copied your comments

Reply:
The whole block of dependencies should go to `openmeetings-parent/pom.xml` (and removed from `core` and `db`)
this way they be available for all modules :)))

Reply:
done